backend/posix/posix.go

```diff
@@ -102,11 +102,11 @@ const (
 )
 
 type PosixOpts struct {
```

Comment:
Some of these structs like this one happen exactly once. Not sure if this is very important to optimize.